core-ui/src/main/java/org/woowatechcamp/banchan/core_ui/utils/DialogUtil.kt

```diff
@@ -0,0 +1,21 @@
+package org.woowatechcamp.banchan.core_ui.utils
+
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import com.google.android.material.dialog.MaterialAlertDialogBuilder
+
+object DialogUtil {
+    inline fun <reified T : Activity> showCartAdditionConfirmPopup(context: Context) {
+        MaterialAlertDialogBuilder(context)
+            .setMessage("선택한 상품이 장바구에 담겼습니다.")
+            .setNegativeButton("장바구니 확인") { dialog, _ ->
+                context.startActivity(Intent(context, T::class.java))
+                dialog.dismiss()
+            }
+            .setPositiveButton("계속 쇼핑하기") { dialog, _ ->
+                dialog.dismiss()
+            }
+            .show()
+    }
+}
```

Comment:
확장함수 형식으로 표현해도 간결할 것 같네요!

```suggestion
inline fun <reified T : Activity> Context.showCartAdditionConfirmPopup() {
        MaterialAlertDialogBuilder(this)
            .setMessage("선택한 상품이 장바구에 담겼습니다.")
            .setNegativeButton("장바구니 확인") { dialog, _ ->
                context.startActivity(Intent(this, T::class.java))
                dialog.dismiss()
            }
            .setPositiveButton("계속 쇼핑하기") { dialog, _ ->
                dialog.dismiss()
            }
            .show()
}
```

Reply:
좋네요 반영하겠습니다!!